src/vs/workbench/contrib/webview/electron-browser/webviewElement.ts

```diff
@@ -231,9 +231,11 @@ export class ElectronWebviewBasedWebview extends BaseWebview<WebviewTag> impleme
 			return;
 		}
 
-		// Clear the existing focus first.
+		// Clear the existing focus first if not already on the webview.
 		// This is required because the next part where we set the focus is async.
-		if (document.activeElement && document.activeElement instanceof HTMLElement) {
+		if (document.activeElement && document.activeElement instanceof HTMLElement && document.activeElement?.tagName !== 'WEBVIEW') {
```

Comment:
nit pick: null propagation on the last check if not required as documet.activeElement is guaranteed to exist, due to first check

Reply:
Good point. Will fix